Kanvas.podspec

```diff
@@ -1,5 +1,5 @@
 Pod::Spec.new do |spec|
-  spec.name         = "KanvasCamera"
+  spec.name         = "Kanvas"
   spec.version      = "1.0.1"
```

Comment:
Do we need to bump the version too since 1.0.1 already exists? Or you do have in mind to delete that tag?

Reply:
Yep!

@danielebogo Do you think this should be `1.1.0` since it will break the API (with type name changes)? It doesn't _technically_ matter since Tumblr is pegged to a tag.

Reply:
If this affects API then I would say 1.1.0